frontend/models/feedback-task-model/dataset-filter/DatasetFilter.model.js

```diff
@@ -0,0 +1,20 @@
+import { Model } from "@vuex-orm/core";
+
+class DatasetFilter extends Model {
+  static entity = "datasetFilters";
+
+  static fields() {
+    return {
+      id: this.uid(),
+      name: this.attr(null),
+      dataset_id: this.attr(null),
+      value: this.attr(null),
+      options: this.attr([]),
+      placeholder: this.string(null).nullable(),
+      component_type: this.string(null).nullable(),
+      group_id: this.string(null).nullable(),
+    };
```

Comment:
add an order attribut, and then query the filters by datasetId and order to always get the filters in the good order